src/components/books/BookInfo.vue

```diff
@@ -14,7 +14,7 @@
         target="_blank"
         rel="noopener"
         data-cy="book-title"
-        @click="$emit('book-title-click')"
+        @click="clickBook(item)"
```

Comment:
Since we pass the book as a prop, we don't need to pass arguments to the function. It could be `@click="clickBook"` directly.